catalog/dags/data_refresh/data_refresh_task_factory.py

```diff
@@ -44,23 +44,20 @@
 - [[Feature] Data refresh orchestration DAG](
```

Comment:
Aside from the addition of the filtered index creation wait task, the changes in this module are almost all related to the abstractions necessary to share ingestion server communication tasks with the new DAGs.